README.md

```diff
@@ -7,22 +7,301 @@ Serlio requires so-called rule packages (RPK) as input, which are authored in Ci
 Serlio is well suited for managing the procedural generation of architectural 3D content in digital sets. However, Serlio is restricted to the procedural generation of single buildings / objects. Serlio does not include the city layouting and street network editing tools of CityEngine i.e. the rich CityEngine toolset to design a city from scratch (or based on geographic data) is still needed.
 
 
-## Documentation
+## Quick Start
+1. Use CityEngine to [export a Rule Package](https://doc.arcgis.com/en/cityengine/latest/help/help-rule-package.htm) (RPK).
+1. Acquire the Serlio binaries, either: 
+    * ... download the [desired release](https://github.com/Esri/serlio/releases)
+    (on linux you still need to manually [install the local build](#linux-2))
+    * ... or [build](#build-instructions) them yourself, which will result in an `install` directory
+1. Start Maya (Note: you may want to start maya from a shell to see the Serlio log output).
+1. Open the plugin manager: "Windows" > "Settings/Preferences" > "Plug-in Manager"
+1. Enable `serlio.so` / `serlio.mll`
+1. The plugin should load and a new menu item `Serlio` should appear in Maya.
+1. In Maya, select a mesh and use the `Serlio` menu to assign the RPK. This will apply the selected rule to each face in the mesh.
```

Comment:
"... to assign a rule package (RPK), for example [this one](link to attached file) which creates a simple extrusion based on a "height" parameter. You can author your own RPKs with CityEngine."

The sample extrude.rpk should have a single "height" attribute and be created with CityEngine 2021.1

Reply:
done